Tests/configure_and_test_integration_instances.py

```diff
@@ -0,0 +1,472 @@
+    module_instance = set_integration_instance_parameters(integration_configuration, integration_params,
+                                                          integration_instance_name, is_byoi)
+    return module_instance
+
+
+def get_new_and_modified_integrations(git_sha1):
+    '''Return 2 lists - list of new integrations and list of modified integrations since the commit of the git_sha1'''
+    # get changed yaml files (filter only added and modified files)
+    tag = get_last_release_version()
+    file_validator = FilesValidator()
+    change_log = run_command('git diff --name-status {}'.format(git_sha1))
+    modified_files, added_files, removed_files, old_format_files = file_validator.get_modified_files(change_log, tag)
+    all_integration_regexes = YML_INTEGRATION_REGEXES
+    all_integration_regexes.extend([INTEGRATION_REGEX, PACKS_INTEGRATION_REGEX, BETA_INTEGRATION_REGEX])
+    added_integration_files = [
+        file_path for file_path in added_files if checked_type(file_path, all_integration_regexes)
+    ]
+    modded_integration_files = [
+        file_path for file_path in modified_files if
+        isinstance(file_path, str) and checked_type(file_path, all_integration_regexes)
+    ]
+
+    new_integrations_names = []
+    for integration_file_path in added_integration_files:
+        integration_yaml = get_yaml(integration_file_path)
+        integration_name = integration_yaml.get('name')
+        if integration_name:
+            new_integrations_names.append(integration_name)
+    modded_integrations_names = []
+    for integration_file_path in modded_integration_files:
+        integration_yaml = get_yaml(integration_file_path)
+        integration_name = integration_yaml.get('name')
+        if integration_name:
+            modded_integrations_names.append(integration_name)
+    return new_integrations_names, modded_integrations_names
+
+
+def is_content_updating(server, username, password):
+    '''Configure Demisto Client and make request to check if content is updating'''
+    # Configure Demisto Client
+    c = demisto_client.configure(base_url=server, username=username, password=password, verify_ssl=False)
+
+    msg = '\nMaking "Get" request to server - "{}" to check if content is installing.'.format(server)
+    print(msg)
+
+    # make request to check if content is updating
+    response_data, status_code, _ = demisto_client.generic_request_func(self=c, path='/content/updating',
+                                                                        method='GET', accept='application/json')
+
+    if status_code >= 300 or status_code < 200:
+        result_object = ast.literal_eval(response_data)
+        message = result_object.get('message', '')
+        msg = "Failed to check if content is installing - with status code " + str(status_code) + '\n' + message
+        print_error(msg)
+        return 'request unsuccessful'
+    else:
+        return response_data
+
+
+def get_content_installation(client):
```

Comment:
not too good of a name, maybe `get_content_version_details`
add docsring about the args, returns sections pls